docs/part-4-labeling-the-data-and-retrain/chapter-20-link-the-model-to-label-studio.md

```diff
@@ -0,0 +1,260 @@
+# Chapter 20 - Link the model to Label Studio
+
+## Introduction
+
+In this chapter, we will start by creating an API for the model using
+[FastAPI](https://fastapi.tiangolo.com). We will then link the model we trained
```

Comment:
Out of topic, but I just realized BentoML uses [startlette](https://www.starlette.io/) internally, not FastAPI. They both are based on the same framework though.